static/index.html

```diff
@@ -7,6 +7,7 @@
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
    <link rel="stylesheet" href="css/style.css">
    <script src="script/mobile-menu.js"></script>
+   <script defer src="/static/fontawesome/fontawesome-all.js"></script>
```

Comment:
What is "defer"? Should we use a slash on the other scr too? See line above (line 9).